.github/workflows/build-images-ci.yaml

```diff
@@ -109,7 +109,7 @@ jobs:
             quay.io/${{ github.repository_owner }}/${{ matrix.name }}-ci:v1.11-race
             quay.io/${{ github.repository_owner }}/${{ matrix.name }}-ci:${{ steps.tag.outputs.tag }}-race
           build-args: |
-            BASE_IMAGE=quay.io/cilium/cilium-runtime:3e2ea3c996d8372ba65ee14058a716f797c67496@sha256:f5ea135fce72807158e520d1754e826b138cf9e97adde528fd992b3e79ef6ee8
+            BASE_IMAGE=quay.io/cilium/cilium-runtime:805506ff0dc0a9886fc1a8ce53191275f8baf62e
```

Comment:
Wrong click sorry => I think there might be an issue here with the image reference as we're missing the `sha256` pin.

Reply:
I'm following steps here: https://docs.cilium.io/en/stable/contributing/development/images/#update-cilium-builder-and-cilium-runtime-images

I think they images are build in the build deployment and running the scripts will add their checksums.